src/features/applicationMetadata/ApplicationMetadataProvider.tsx

```diff
@@ -24,7 +33,39 @@ const { Provider, useCtx, useLaxCtx, useHasProvider } = delayedContext(() =>
   }),
 );
 
-export const ApplicationMetadataProvider = Provider;
+function VerifyMinimumVersion({ children }: PropsWithChildren) {
+  const { altinnNugetVersion } = useApplicationMetadata();
+
+  if (!altinnNugetVersion) {
+    return <OldVersionError minVer={MINIMUM_APPLICATION_VERSION.name} />;
+  }
+
+  let isMinimumVersion = false;
```

Comment:
Would love for this to be a util function. `isNewerVersion` for instance :heart_eyes:

Reply:
Pretty much nothing will run now if the version is older than the minimum - we'll just fetch `applicationmetadata.json` and then fail with this error message immediately after. So I don't think there's much 'utility' in having a function that returns `true` in all code except rare cases that are stopped by `index.tsx`. :sweat_smile:

Reply:
I was thinking that you create a function `isNewerThat(v1,v2)` which could be used inside `ApplicationMetadataProvider` like so: `isNewerThan(minimumVersion, currentVersion)` 

Not important of course, since it's only used one place, and it was just a suggestion on organizing the code. But the next time someone needs to compare a version number, they could find that function which explains what it does. By implementing the logic inside `ApplicationMetadataProvider`, the risk of someone implementing the same functionality twice increases.

Reply:
Ahhh, right, yep. For once we're actually beaten by PHP which [has this built-in](https://www.php.net/manual/en/function.version-compare.php)! Kinda missed that function. Agree with you that a proper version compare function/utility would be preferable, yep. I'm rushing to get this out at the moment, but I'll make a note of this. :pray: